docs/src/main/sphinx/admin/properties-exchange.md

```diff
@@ -39,7 +39,7 @@ network utilization.
 
 - **Type:** {ref}`prop-type-string`
 - **Allowed values:** `NONE`, `LZ4`, `ZSTD`
-- **Default value:** `NONE`
+- **Default value:** `LZ4`
```

Comment:
Also delete the "Defaults to..." below since it is redundant.

Reply:
And confirm in the code

Reply:
From what I have found, the default is LZ4.